ContextDesktop/src/main/java/com/github/migi_1/Context/model/MainEnvironment.java

```diff
@@ -165,12 +165,12 @@ private void checkPathCollision() {
         for (Carrier carrier : platform.getCarriers()) {
             if (boundingBoxWallLeft.intersects(carrier.getModel().getWorldBound())) {
                 commander.move(new Vector3f(0, 0, -0.3f));
-                platform.move(new Vector3f(0, 0, -0.3f));                
-                carrier.move(new Vector3f(0, 0, -0.3f));                
+                platform.move(new Vector3f(0, 0, -0.3f));
+                carrier.move(new Vector3f(0, 0, -0.3f));
             }
             else if (boundingBoxWallRight.intersects(carrier.getModel().getWorldBound())) {
                 commander.move(new Vector3f(0, 0, 0.3f));
-                platform.move(new Vector3f(0, 0, 0.3f));                
+                platform.move(new Vector3f(0, 0, 0.3f));
                 carrier.move(new Vector3f(0, 0, 0.3f));
             }
         }
```

Comment:
Since these vectors all seem the same, maybe you can create two different attributes for these vectors?